Component/SolidMechanics/FEM/HyperElastic/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 3.12)
+project(Sofa.Component.SolidMechanics.FEM.HyperElastic LANGUAGES CXX)
+
+set(SOFACOMPONENTSOLIDMECHANICSFEMHYPERELASTIC_SOURCE_DIR "src/sofa/component/solidmechanics/fem/hyperelastic/")
+
+set(HEADER_FILES
+    ${SOFACOMPONENTSOLIDMECHANICSFEMHYPERELASTIC_SOURCE_DIR}/config.h.in
+    ${SOFACOMPONENTSOLIDMECHANICSFEMHYPERELASTIC_SOURCE_DIR}/init.h
+    ${SOFACOMPONENTSOLIDMECHANICSFEMHYPERELASTIC_SOURCE_DIR}/BaseMaterial.h
```

Comment:
I think it would make sense to put all material classes into a dedicated folder, such as `${SOFACOMPONENTSOLIDMECHANICSFEMHYPERELASTIC_SOURCE_DIR}/Materials`

Reply:
yessir